### ISSUE TITLE
RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stderr maxBuffer length exceeded

### ISSUE THREAD

**Issue by azu** [nicojs/node-install-local#8]
**Env:**

```
node -v
v10.2.1
npm -v
6.1.0
```

I've tried to install-local for large repository, I met following error.

```
[install-local] packing - 0/1{ RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stderr maxBuffer length exceeded
    at Socket.onChildStderr (child_process.js:362:14)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:279:12)
    at readableAddChunk (_stream_readable.js:260:13)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at Pipe.onread (net.js:636:20)
  cmd:
   'npm pack /my/large/repo' }
```

- [Errors | Node.js v10.4.1 Documentation](https://nodejs.org/api/errors.html#errors_err_child_process_stdio_maxbuffer)

` child_process.exec`'s `maxBuffer` is `200 * 1024` by default.

- [Child Process | Node.js v10.4.1 Documentation](https://nodejs.org/api/child_process.html#child_process_maxbuffer_and_unicode)

Can I avoid this error?

**Comment by nicojs** [comment]
Thanks for reporting this issue.

I think you're right. I'm using the default buffer size. I suggest setting it to 10MB (`1024*1024*10`)

Code is here:
https://github.com/nicojs/node-install-local/blob/047e35dd6653d69464c8bf1cd23fad4c27e51c3f/src/LocalInstaller.ts#L115

And here: 
https://github.com/nicojs/node-install-local/blob/047e35dd6653d69464c8bf1cd23fad4c27e51c3f/src/LocalInstaller.ts#L78

I'm happy to accept a PR.

**Comment by nicojs** [comment]
Released in 0.6.0
